doc/manual/templates/default/keywords.lst

```diff
@@ -20,11 +21,13 @@ $Dynamic
 $Include
 $Lang
 $Static
+Abstract
 Assert
 AssertWarn
 Abs
 Access
 Acos
+Add
```

Comment:
`add` isn't an FB keyword, or is it?

Reply:
Add is a graphics keyword documented at https://www.freebasic.net/wiki/wikka.php?wakka=KeyPgAddGfx

Reply:
Oh, oops, missed it.